README.de.md

```diff
@@ -0,0 +1,160 @@
+
+Zum Beispiel begann der populäre Hersteller `Caputo` erst 1924 damit, sein Weizenmehl zu verkaufen. Neue Arten von Mehl wurden mithilfe von Radioaktivität oder Toxinen entwickelt, was zu schnelleren, genetischen Mutationen und damit höheren Ernteerträgen geführt hat. Daher kann auch diskutiert werden, dass bei der Erfindung der Pizza bereits neuartige Mehlarten und morderne Hefe verfügbar waren. Hefe macht das Pizzamachen zudem einfacher. Zum Beispiel kann zu lange fermentierter Sauerteig einen negativen Einfluss auf die Glutenstruktur des Teigs haben, was sich bei Verwendung von Hefe leichter kontrolliert werden kann.
+
+Traditionellerweise wird die neapolitanische Pizza bereits am Tag vorher vorbereitet. Dies ist eine der wichtigsten Voraussetzungen, um einen erstklassigen Teig zu machen, der auch zu Brot verarbeitet gut schmecken würde. Eine Fermentierung bei Raum- und kalten Temperaturen sorgt zudem für die Bildung von speziellen Enzimen, die den komplexen aber angenehmen Geschmack von Pizzateig ausmachen.
+
+## Zutaten
+
+Für den unerfahrenen Pizzabäcker ist es häufig überraschend, wie wenig Zutaten für einen Pizzateig nötig sind. Die Menge der Zutaten basiert auf der Menge des Weizenmehls, die verwendet wird:
+
+* Weizenmehl
+* 65% warmes Wasser
+* 2% Salz
+* 0,05% trockene oder 0,15% frische Hefe
+
+Die Menge an Wasser, Salz und Hefe in Prozent relativ zum Mehl anzugeben, macht es einfacher, das Rezept für verschiedene Anzahlen von Pizzen anzupassen. Dieses Pinzip nennt sich auch "Teigausbeute" und macht es einfach, die Menge des gewünschten Teigs zu skalieren.
+
+Angenommen wir wollen zwei kleine Pizzas mit 200g Mehl machen. Dann würden die Zutaten sich wie folgt zusammensetzen (keine Sorge ein Rechner folgt später):
+
+* 200g Mehl
+* 130ml warmes Wasser
+* 4g Salz
+* 0,1g trockene oder 0,3g frische Hefe
+
+Eine typische Pizza in Napoli hat eine Teigmasse von ungefähr 250g. Allerdings sollte man beachten, dass die Pizzaöfen in Napoli auch sehr groß sind. Ich schlage deshalb für den häuslichen Gebrauch vor, die Teigmenge auf ca. 220g zu reduzieren. Im Bild oben mache ich eine solche 220g-Pizza.
+
+Um die Berechnung der Zutatenmengen zu vereinfachen, habe ich einen [kleinen Rechner](https://pizza-calculator.the-bread-code.io) erstellt (Webseite ist auf englisch).
+
+## Das richtige Mehl
+
+Das Mehl die wichtigste Zutat des Teigs. Daher dient hier ein eigenes Kapitel dazu, das richtige Mehl zu finden. Erfahrene Pizzabäcker können einige Defizite von schlechtem Mehl umgehen. Für Einsteiger kann das Mehl allerdings der Unterschied zwischen einer grandiosen Pizza oder eines kompletten Reinfalls sein.
+
+Es gibt aber eine Fausregel: **Nimm' ein Mehl mit hohem Proteinanteil.**
```

Comment:
`Faustregel`

Reply:
Erledigt 👌 

In Zeile 39 fehlte tatsächlich auch noch ein "ist" 😅